contracts/Transfer.sol

```diff
@@ -95,26 +95,41 @@ contract Transfer is FraudProofHelpers {
     function processTransferBatch(
         bytes32 stateRoot,
         bytes memory txs,
-        Types.AccountMerkleProof[] memory accountProofs
+        Types.AccountMerkleProof[] memory accountProofs,
+        uint256 tokenType,
+        uint256 feeReceiver
```

Comment:
We require the transactions in this commitment to have the same tokenType, so that we can guarantee the fee receiver receives the same types of the token.